src/main/java/com/microsoft/sqlserver/jdbc/Util.java

```diff
@@ -373,9 +373,14 @@ else if (ch == ':')
                         name = SQLServerDriver.getNormalizedPropertyName(name, logger);
                         if (null != name) {
                             if (logger.isLoggable(Level.FINE)) {
-                                if ((false == name.equals(SQLServerDriverStringProperty.USER.toString()))
-                                        && (false == name.equals(SQLServerDriverStringProperty.PASSWORD.toString())))
```

Comment:
Originally, the condition checked if we are working with the database password and then decides not to log it.

Now, we check if the property name includes the string "password" so the first if condition is not needed anymore